datasets/refresd/refresd.py

```diff
@@ -0,0 +1,109 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Rationalized English-French Semantic Divergences (REFreSD) dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{briakou-carpuat-2020-detecting,
+    title = "Detecting Fine-Grained Cross-Lingual Semantic Divergences without Supervision by Learning to Rank",
+    author = "Briakou, Eleftheria and Carpuat, Marine",
+    booktitle = "Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)",
+    month = nov,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.emnlp-main.121",
+    pages = "1563--1580",
+}
+"""
+
+_DESCRIPTION = """\
+The Rationalized English-French Semantic Divergences (REFreSD) dataset consists of 1,039 
+English-French sentence-pairs annotated with sentence-level divergence judgments and token-level 
+rationales. For any questions, write to ebriakou@cs.umd.edu.
+"""
+
+_HOMEPAGE = "https://github.com/Elbria/xling-SemDiv/tree/master/REFreSD"
+
+_LICENSE = ""
```

Comment:
No license in the repo, I could open a PR with MIT license, I guess it would get merged.

Reply:
Hi Manuel :) I just merged your request.

Reply:
Hey, thanks a lot for your message :upside_down_face: 
I'll get back to it, probably tomorrow.